tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,92 @@
+/* bender-tags: a11ychecker,inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+/* bender-include: ../balloonpanel/_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	CKEDITOR.dom.window.prototype.getViewPaneSize = function() {
+		return {
+			width: 1183,
+			height: 1818
+		};
+	};
+
+	bender.editor = {
+		name: 'editor1',
+		config: {
+			allowedContent: true,
```

Comment:
I don't think there's a strong reason to disable ACF.

Reply:
It was the shortest way to make `<span id="marker">bom</span>` available